src/data/modelMetadata.ts

```diff
@@ -0,0 +1,59 @@
+export type Model = {
+  id: string;
+  name: string;
+  apiEndpoint: string;
+  pythonExampleEndpoint: string;
+  javascriptExampleEndpoint: string;
+  code: string;
+};
+
+const regmbModel: Model = {
+  id: "rembg",
+  name: "Rembg",
+  apiEndpoint: "https://103961668-rembg.gateway.alpha.fal.ai",
```

Comment:
move this to public ENV var?

Reply:
the whole thing or just the api endpoint part?